### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by nikihowe**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

The readme was out of date. This starts to address that (more work to be done on this soon).

## Checklist

- [ ] The documentation is up-to-date with the changes I made.
- [ ] I have read the **CONTRIBUTING** document and completed the CLA (see **CONTRIBUTING**).
- [ ] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,14 @@
...
+To get the code up and running, see the dockerfile and instructions in https://github.com/AlignmentResearch/polygames-scripts.
 
-cmake .. -DCMAKE_BUILD_TYPE=relwithdebinfo -DPYTORCH15=ON
-make -j
-
-```
+## Checkpoints
 
-Ludii support can be disabled by appending `-DWITH_LUDII=OFF` to the cmake command (required if you don't have jdk)
+http://dl.fbaipublicfiles.com/polygames/checkpoints/list.txt.
```
Comment (tomtseng) [inline README.md]: might want to elaborate on what this is, it's not immediately clear without context

File: README.md [hunk 2]
```diff
@@ -1,58 +1,14 @@
-[![CircleCI](https://circleci.com/gh/facebookincubator/Polygames.svg?style=svg)](https://circleci.com/gh/facebookincubator/Polygames)
```
Comment (tomtseng) [inline README.md:1]: eventually it'd be good to get CI running again, but I guess there's not much to test (besides that the Docker image builds?) until we get the original polygames tests compiling again

File: README.md [hunk 3]
```diff
@@ -1,58 +1,14 @@
...
+This repository is a fork of https://github.com/facebookarchive/Polygames.
 
-```
-git clone --recursive https://github.com/facebookincubator/polygames
-cd polygames
+## Installation
 
-mkdir build
-cd build
+To get the code up and running, see the dockerfile and instructions in https://github.com/AlignmentResearch/polygames-scripts.
```
Comment (tomtseng) [inline README.md:7]: hmm why are the Dockerfile and scripts in a separate repo instead of in subdirectories of this repo?

**Review by tomtseng — APPROVED** [review]

**Review by tomtseng — APPROVED** [review]

**Comment by nikihowe** [comment]
Note that we should still keep an open mind as to where the Dockerfile lives. It should be easy to change though, so it's not a blocker for the merge as far as I can tell.

**Pull request merged**